pkg/dynamicinstrumentation/testutil/sample/kafka_go_forwarder/main.go

```diff
@@ -0,0 +1,166 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/datastreams"
+
+	"github.com/Shopify/sarama"
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
```

Comment:
These are net-new imports for the agent. Is there a way to write this test program using `github.com/twmb/franz-go`?